src/screens/Embed/Embed.tsx

```diff
@@ -94,7 +94,9 @@ export default function Embed({
   isRiskMap?: boolean;
 }) {
   const navigateToCAN = () => {
-    window.top.location.href = 'https://covidactnow.org/';
+    if (window.top) {
```

Comment:
TS was complaining that `window.top` could be undefined, so check that it exists